build_library/grub_install.sh

```diff
@@ -183,6 +183,12 @@ sudo grub-mkimage \
     --output "${ESP_DIR}/${GRUB_DIR}/${CORE_NAME}" \
     "${CORE_MODULES[@]}"
 
+if [[ "${FLAGS_target}" != i386-pc ]]; then
```

Comment:
Since they become part of the `core.img` it should be possible to remove this, or? I think we can try this out and either BIOS will boot or not.

Reply:
I will also try this out and see if BIOS works or not.

Reply:
With @pothos suggestion to remove also the amd64 parts:

```
used 61681
free 67358

versu before:

used 61782
free 67258
```